### PR TITLE
Fix bsc1165587 and other minor changes

### PR DESCRIPTION
**Pull request by MalloZup**
# desc:

- this patch backport the change/patch, and also adapt the approach for different binaries location, since we need to do this for different OS version which might use diff binaries.

- it removes devel gems which aren't needed.

# note for reviewers:

I have already tested this and worked well. I will rebuild the pkg and retest it just in case. (that's why wip) but is already reviewable

### CODE REVIEW

File: hawk/config/application.rb [hunk 1]
```diff
@@ -46,5 +46,24 @@ class Application < Rails::Application
...
+      [
+        "/usr/libexec/pacemaker",
+        "/usr/lib64/pacemaker",
+        "/usr/lib/pacemaker",
+        "/usr/lib64/heartbeat",
+        "/usr/lib/heartbeat"
+      ].each do |dir|
+        [
+          "crmd",
+          "pacemaker-controld",
```
Comment (MalloZup) [inline hawk/config/application.rb]: @diegoakechi  @petrpavlu @gao-yan I was wondering if `pacemaker-controld` is not backward compatible

Comment (gao-yan) [reply hawk/config/application.rb]: Although the names of the daemons have been changed from SLE15 SP1 as of:

https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Daemon_Changes

, we still maintain the old names as symbol links to the new executables.

Other than names, the supported options remain the same. 

AFAIK, what hawk and crmsh are concerned about is invoke these daemons' `metadata` command and retrieve the metadata. I'm pretty sure this still works all the same way.

Comment (MalloZup) [reply hawk/config/application.rb]: nice thx lot @gao-yan

File: hawk/Gemfile [hunk 2]
```diff
@@ -26,15 +26,7 @@ gem "gettext_i18n_rails", ">= 1.8"
...
-
-group :development do
-  source "https://rubygems.org"
-  gem "web-console", ">= 3.5"
-  gem "spring", ">= 2.0"
-  gem "uglifier", ">= 3"
-  gem "gettext", ">= 3.2", require: false
-  gem "byebug"
-end
+gem "uglifier", ">= 3"
```
Comment (MalloZup) [inline hawk/Gemfile]: https://github.com/ClusterLabs/hawk/issues/187

File: hawk/Gemfile [hunk 3]
```diff
@@ -26,15 +26,7 @@ gem "gettext_i18n_rails", ">= 1.8"
 gem "gettext_i18n_rails_js", ">= 1.3"
 gem "sprockets", ">= 3.7"
 gem "kramdown", ">= 1.14"
-
-group :development do
```
Comment (diegoakechi) [inline hawk/Gemfile]: @MalloZup I don't think it is really needed, as this are gem used only if you run bundler with the development mode... so these are useful for debug or development.. but does not affect HAWK build on the default or production setup.

Comment (MalloZup) [reply hawk/Gemfile]: this is needed because OBS try to build such gems from dev. And we have a dirty patch for this.

**Pull request merged**